odl/operator/pspace_ops.py

```diff
@@ -83,17 +90,11 @@ class ProductSpaceOperator(Operator):
 
         :math:`[\mathcal{A}(x)]_i = \sum_{j=1}^m \mathcal{A}_{ij}(x_j)`.
 
-    Notes
```

Comment:
This is somewhat redundant now that we have the other three operators that cover this case.